package.json

```diff
@@ -16,7 +16,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "deploy": "npm run build && scp -r ./build/* valerievolfman@news-explorer-valerie-v.students.nomoredomainssbs.ru:/home/valerievolfman/frontend"
   },
   "eslintConfig": {
```

Comment:
```jsx

      if (e.target.matches(".popup")) {
        const popup = document.querySelectorAll(".popup");
        if (e.target !== popup) {
          closeAllPopups();
        }
      }

```


* please, don't use `document.querySelector` in React. It was created specially not to search DOM elements directly. It is a slow operation that slows down the app

the overlay handler works without such searches

```jsx


      if (e.target.matches(".popup")) {
        closeAllPopups();
      }

```